test/integration/src/py/test_sifnode_peggy1_pause_unpause.py

```diff
@@ -0,0 +1,114 @@
+from typing import Tuple
+import pytest
+
+import siftool_path
+from siftool import eth, test_utils, sifchain, cosmos
+from siftool.inflate_tokens import InflateTokens
+from siftool.common import *
+from siftool.test_utils import EnvCtx
+
+def test_pause_unpause_no_error(ctx: EnvCtx):
+    res = ctx.sifnode.pause_peggy_bridge(ctx.sifchain_ethbridge_admin_account)
```

Comment:
These return types should be better typed, I have no way of knowing what to expect the return type to look like without running it in a debugger.